### DISCUSSION TITLE
I must be in the wrong place...

### DISCUSSION THREAD

**Discussion by FJL65** [statamic/cms#8668] [Q&A]
Hello, I'm glad to read that the Statamic community are very helpful and working with Statamic is a joy. I'm just an end user trying to maintain content on a site previously established by someone else and finding it difficult. Despite searching high and low I haven't found any documentation that explains simple stuff like the syntax for a link or the way to bring up the media browser to upload documents. Sorry to intrude on all the serious stuff, but if you could point me to some documentation that would be very helpful.
Thanks,
Frank

**Comment by jackmcdade** [comment]
Hey there, this is a pretty good place to start: https://statamic.dev/content-managers-guide
